coderadar-graph/src/main/java/io/reflectoring/coderadar/graph/query/repository/ContributorQueryRepository.java

```diff
@@ -32,7 +32,7 @@
   @Query(
       "MATCH (p)-[:CONTAINS_COMMIT]->(c:CommitEntity) WHERE ID(p) = {0} AND c.name = {1} WITH c LIMIT 1 "
           + "CALL apoc.path.subgraphNodes(c, {relationshipFilter:'IS_CHILD_OF>'}) YIELD node WITH node as c ORDER BY c.timestamp DESC WITH collect(c) as commits "
-          + "CALL apoc.cypher.run('UNWIND commits as c MATCH (c)<-[:CHANGED_IN]-(f) WHERE f.path = {path} RETURN f', {commits:commits, path: {2}}) YIELD value "
+          + "CALL apoc.cypher.run('UNWIND commits as c MATCH (c)<-[:CHANGED_IN]-(f) WHERE f.path = path RETURN f', {commits:commits, path: {2}}) YIELD value "
```

Comment:
Are the brackets unnecessary?

Reply:
In this case, there should be no difference of either using brackets or not.